### PR TITLE
Attempted Week0 Challenge

### PR DESCRIPTION
**Pull request by goodnessemmanuel**

### CODE REVIEW

**Comment by goodnessemmanuel** [comment]
@bondz kindly check my algorithms for your test and/or correction. Thank you, sir.

File: src/brackets/index.js [hunk 1]
```diff
@@ -4,6 +4,44 @@
...
+
+        }
+    }
+    return stackBrackets.length === 0? 'valid' : 'invalid';//use this ternary operator to return correct option
+}
 
 module.exports = isValid;
+console.log(isValid("{}()"));//outputs valid
+console.log(isValid("}{()"));//returns invalid
+console.log(isValid("({}[])"));//returns valid
```
Comment (bondz) [inline src/brackets/index.js:47]: Remove extraneous log statements when submitting your requests.

File: src/roman-numerals/index.js [hunk 2]
```diff
@@ -4,6 +4,66 @@
...
+	for (let letter of romanNumerals)
+	{
+
+		if (romanKeys.indexOf(letter) === -1)
+		{
+            //if any of the letter in the input string parameter is not contained in the roman keys
+            return `invalid roman figure ${letter}`;
+            
+		}
+	}
```
Comment (bondz) [inline src/roman-numerals/index.js:29]: Why not do this check concurrently in the loop below.

File: src/brackets/index.js (hunk 1 above)
Comment (goodnessemmanuel) [reply src/brackets/index.js:47]: Thank you, sir, I will affect the change here and also ensure it in my subsequent challenge

File: src/roman-numerals/index.js (hunk 2 above)
Comment (goodnessemmanuel) [reply src/roman-numerals/index.js:29]: I tried it earlier in the same loop but the program runs in a certain way that doesn't feel right to me. When the invalid roman figure is not at the first index of the user input, the loop still runs to that point before logging the error, but I want the program to terminate irrespective of the previous valid roman figure before it gets to the invalid roman figure within the input string.

**Pull request merged**